omod/src/test/java/org/openmrs/web/controller/ForgotPasswordFormControllerTest.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.web.controller;
+
+import org.openmrs.User;
+import org.openmrs.test.BaseContextSensitiveTest;
+import org.openmrs.web.test.BaseModuleWebContextSensitiveTest;
+import org.openmrs.web.test.BaseWebContextSensitiveTest;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.net.BindException;
+/**
+ * Test the different aspects of
+ * {@link org.openmrs.web.controller.ForgotPasswordFormController}
+ */
+public class ForgotPasswordFormControllerTest extends BaseModuleWebContextSensitiveTest {
+
+	private static Level initialLogLevel;
+
+	private static final String CLASS_TO_LOG = "org.openmrs.api.db.hibernate.HibernateUserDAO";
+
+	protected static final String TEST_DATA= "org/openmrs/web/controller/include/ForgotPasswordFormControllerTest.xml";
+
+	/**
+	 * Check to see if the admin's secret question comes back
+	 *
+	 * @throws Exception
+	 */
+	@Test
+	public void shouldNotNotFailOnNotFoundUsernameTEST() throws Exception {
+		initializeInMemoryDatabase();
+		executeDataSet(TEST_DATA);
+		authenticate();
+
+		ForgotPasswordFormController controller = new ForgotPasswordFormController();
```

Comment:
Because you forgot to change the bean name to fetch from the application context, in fact the error message kind of gives a clue because you are fetching a concept form but expecting to get back a forgot password form